app/models/notification/right_side_popup.rb

```diff
@@ -0,0 +1,9 @@
+class Notification::RightSidePopup < Notification
```

Comment:
As I mentioned before, why we cannot use just <code>Popup</code> ?  and let rightside be the option. We will probably have few kinds of popups in future.